recipes/allennlp/meta.yaml

```diff
@@ -0,0 +1,148 @@
+{% set name = "allennlp" %}
+{% set version = "0.7.2" %}
+
+{% set spacy_model = "en_core_web_sm" %}
+{% set spacy_model_version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ name }}
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: c51d32598ed0efb0599c15f463e2fe792e06d843964528b6bca9747b0d31be5c
+
+  # Spacy Model
+  - folder: {{ spacy_model }}
+    url: https://github.com/explosion/spacy-models/releases/download/{{ spacy_model }}-{{ spacy_model_version }}/{{ spacy_model }}-{{ spacy_model_version }}.tar.gz
+    sha256: 9d9d231d7c9cbad82178b566fdb25768e3d5098af23ce11e03ec4f432f4a2298
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install ./{{ name }} --no-deps -vv"
+  skip: True  # [not linux or py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pytorch >=0.4.1,<0.5.0
+    - jsonnet ==0.10.0  # [not win32]
+    - overrides
+    - nltk
+    - msgpack-python >=0.5.6,<0.6.0
+    - spacy >=2.0,<2.1
+    - numpy
+    - tensorboardx ==1.2
+    - cffi 1.11.*
+    - awscli >=1.11.91
+    - boto3
+    - moto ==1.3.4
+    - flask ==1.0.2
```

Comment:
very strict pins

Reply:
I see where you are coming from, and I would not have done it this way if I wrote the upstream package. However, at the moment this is what they have in their [setup.py](https://github.com/allenai/allennlp/blob/e2766bcee5ac802cc67db1e78f35f736123a89b8/setup.py#L103-L137). 

@CurtLH relaxed the cffi pinning so we could clear the satisfiability with gevent 1.3.6.

Reply:
I reported the issue upstream and cc'ed you on the issue number.  Depending on their updates we can update the pinnings on the feedstock and rebuild.